crates/scroll/consensus/src/curie.rs

```diff
@@ -1,3 +1,22 @@
+//! Curie fork transition for Scroll.
+//!
+//! On block 7096836, Scroll performed a transition to the Curie fork state, which brought  various
+//! changes to the protocol:
+//!    1. Fee reduction cost thanks to the use of compressed blobs on the L1.
+//!    2. Modified [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) pricing
+//!       model along with support for EIP-1559 and EIP-2930 transactions.
```

Comment:
We have support for `EIP-1559` and `EIP-2930`? The docs suggest we don't but they may be outdated https://docs.scroll.io/en/technology/chain/transactions/#transaction-type.

Reply:
As far as I can see [here](https://arc.net/l/quote/ucmdvtxg) yes

Reply:
Opened an issue for the docs https://github.com/scroll-tech/scroll-documentation/issues/407